### PR TITLE
Cards refactor

### PR DESCRIPTION
**Pull request by Trombach**
In this PR:
* `app-cards` has been replaced by new, content-specific cards
* re-factor of content pages, removing some templates
*  added typescript types for cypress e2e tests

### CODE REVIEW

File: research-hub-web/src/app/components/articles/articles.component.html [hunk 1]
```diff
@@ -1,116 +1,201 @@
...
+          fxLayoutGap.lt-lg="25px"
+          fxLayout.lt-sm="column"
+          fxLayoutGap.lt-sm="0px"
+        >
+          <app-standard-card
+            *ngFor="let content of article.relatedItemsCollection.items"
+            [contentItem]="content"
+          ></app-standard-card>
         </div>
-    <div class="section"></div>
```
Comment (rosemcc) [inline research-hub-web/src/app/components/articles/articles.component.html:115]: The section class adds some top and bottom margin - looks like it's missing from articles (and maybe other page types?) now - see the bottom of the page on mobile view. Just need to add that margin back in somewhere.

File: research-hub-web/src/app/components/casestudys/case-study.component.html [hunk 2]
```diff
@@ -1,115 +1,173 @@
...
+    </div>
+  </div>
+
+  <div class="site-padding" ngClass.lt-md="site-padding-mobile">
+    <app-breadcrumbs
+      [contentItem]="parentSubHubs"
+      [title]="caseStudy.title"
+    ></app-breadcrumbs>
+
+    <div fxLayout="row" fxLayoutAlign="space-evenly" fxLayout.lt-lg="column">
```
Comment (rosemcc) [inline research-hub-web/src/app/components/casestudys/case-study.component.html:66]: In the three column layout (same on articles) might just need a small amount of margin added to the right of the body text. It's almost touching the cards at ~1300px screen width

File: research-hub-web/src/app/components/fundings/fundings.component.html [hunk 3]
```diff
@@ -1,192 +1,257 @@
...
-                <div *ngIf="funding.callToAction">
-                    <a href="{{ funding.callToAction }}" target="_blank" mat-flat-button class="standard-button section">
-                        <span *ngIf="funding.callToActionLabel">{{ funding.callToActionLabel }}</span>
-                        <span *ngIf="!funding.callToActionLabel">View Funding</span>
-                    </a>
-                </div>
-                <div class="mini-hr" *ngIf="!funding.callToAction"><hr/></div>
-            </div>
-
-            <!-- Funding Purpose -->
```
Comment (rosemcc) [inline research-hub-web/src/app/components/fundings/fundings.component.html:113]: Missed this section! - Funding purpose is another rich text field at the start of funding pages

File: research-hub-web/src/app/components/home/featured/featured.component.html [hunk 4]
```diff
@@ -1,8 +1,9 @@
-<div *ngIf="featuredItems$ | async; let featuredItems">
-  <app-content-container>
-    <app-content-title [title]="title"></app-content-title>
-    <ng-container *ngIf="featuredItems.items.length > 0">
-      <app-cards [flex]="'row wrap'" [contentItem]="featuredItems"></app-cards>
-    </ng-container>
-  </app-content-container>
-</div>
+<app-content-container *ngIf="featuredItems$ | async as featuredItems">
```
Comment (rosemcc) [inline research-hub-web/src/app/components/home/featured/featured.component.html:1]: This looks much better! Just might be worth adding a bit of margin/gap between items as they are touching at certain screen widths (see at ~1236px)

File: research-hub-web/src/app/components/casestudys/case-study.component.html [hunk 5]
```diff
@@ -1,115 +1,173 @@
...
-
-    <ng-template #caseStudyBody>
-        <ngx-contentful-rich-text id="bodyMediaList" class="section-2" style="word-break:break-word;"
-            [document]="caseStudy.bodyText?.json"
-            [nodeRenderers]="nodeRenderers"
-            [markRenderers]="markRenderers"
-        ></ngx-contentful-rich-text>
-    </ng-template>
-
-    <ng-template #caseStudyReferences>
```
Comment (rosemcc) [inline research-hub-web/src/app/components/casestudys/case-study.component.html:23]: Missing this References section. It should go after the body text :)

**Review by rosemcc — COMMENTED** [review]
Awesome work, Lukas. Just a few minor things to address but otherwise it's looking great!!

File: research-hub-web/src/app/components/articles/articles.component.html (hunk 1 above)
Comment (Trombach) [reply research-hub-web/src/app/components/articles/articles.component.html:115]: Ahh that's what this was for. I'm going to add the ```margin-bottom``` class to the top level div in each content page

**Pull request merged**